app/views/dresses/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= simple_form_for(@dress) do |f| %>
+  <%= f.input :brand %>
+  <%= f.input :color %>
+  <%= f.input :size %>
+  <%= f.input :style %>
+  <%= f.input :description %>
+  <%= f.input :price_per_day %>
+  <%= f.input :photos, as: :file, input_html: { multiple: true } %>
+<% end %>
```

Comment:
I think we need a submit here right? 
    <%= f.submit "Add dress", class: "btn btn-primary" %>

Reply:
Oh wow I forgot the submit button! Thanks for finding that.